xml/slurm.xml

```diff
@@ -887,6 +887,19 @@ Description="subgroup" Organization=bavaria</screen>
     upgrades are not installed unintentionally and gives you the opportunity
     to plan version upgrades beforehand.
    </para>
+   <important>
+     <title><command>zypper up</command> is not recommended</title>
+     <para>
+       On systems running &slurm;, updating packages with <command>zypper up</command>
+       is not recommended. <command>zypper up</command> attempts to update all installed
+       packages to the latest version, so might install new versions of &slurm; outside of
```

Comment:
One small clarification:
... might install a new major version of &slurm;

Please make sure &slurm; expands to 'Slurm' - it is no longer considered to be an acronym but a word. The Schedmd folks were quite adamant about it on SC23.

Reply:
Thank you, I'll make that change.  

I can confirm that &slurm; expands to "Slurm" :+1: